client/src/components/index.js

```diff
@@ -7,6 +7,12 @@ import RadioButton from "./radiobuttons";
 import Select from "./select";
 import SideNav from "./sidenav";
 import TimeButton from "./timebutton";
+import CustomerPageNavBar from "./CustomerPageNavBar";
```

Comment:
You have `CustomerPageNavBar` and then you have `Checkbox` ...etc, but before you have `header` and `durationcard`... do you start your folders with upper or lowercase? And are they using camelCase or just lowercase?